test_util/compareECL.cpp

```diff
@@ -60,6 +61,29 @@ static void printHelp() {
 }
 
 
+int count_files_case(const std::string rootName){
```

Comment:
You could mark the `count_files_case()` function as `static` (or placed in an anonymous namespace)&mdash;mostly to avoid diagnostics concerning "missing declarations".